src/core_scan/Core_scan.ml

```diff
@@ -1017,11 +1017,15 @@ let scan ?match_hook config ((valid_rules, invalid_rules), rules_parse_time) :
 (* Entry point *)
 (*****************************************************************************)
 
+let get_rules config =
+  Common.with_time (fun () -> rules_from_rule_source config)
+[@@trace]
```

Comment:
maybe @@tracing, to imitate @@profiling?

Reply:
Other ppxes are usually verbs (`show`, `compare`), so  thought `trace` would be more consistent. But I don't have a very strong preference